### PR TITLE
Tests fix

### PR DESCRIPTION
**Pull request by Dzeranov**
### Motivation and context

A lot of tests were broken when auth system was changed. So this PR is basically a stabilizer for tests.

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

**Review by vkomodey — CHANGES_REQUESTED** [review]

File: tests/cypress/support/commands.js [hunk 1]
```diff
@@ -13,73 +13,77 @@ require('../plugins/compareImages/compareImagesCommand');
...
+            signed_email: signedEmail,
+        },
+    }).then((response) => {
+        authKey = response['body']['key'];
+    });
+    cy.visit('/', {
+        headers: {
+            Authorization: `Token ${authKey}`,
+        },
+    });
```
Comment (vkomodey) [inline tests/cypress/support/commands.js:56]: are you sure that this is working? the whole construction

`cy.request({}).then(() => {})` is asyncronus. But then you are doing `cy.visit(..)` which should wait for the `authKey` fulfillment. So I'm thinking that this `authKey` should be `undefined` at this moment

AND:

no need to create global var here. just push it to `then` pipeline:

```
cy.request({})
    .then(response => response['body']['key'])
    .then(token => cy.visit('/', {.. // expression with auth key}
```

Comment (vkomodey) [reply tests/cypress/support/commands.js:56]: or cypress requests are synchronous?

File: tests/cypress/integration/actions_users/registration_involved/case_4_assign_taks_job_users.js [hunk 2]
```diff
@@ -21,44 +21,69 @@ context('Multiple users. Assign task, job.', () => {
...
+            },
+        }).then(async (response) => {
+            authKey = await response['body']['key'];
+            cy.visit('/', {
+                headers: {
+                    Authorization: `Token ${authKey}`,
+                },
+            });
+        });
+    }
```
Comment (vkomodey) [inline tests/cypress/integration/actions_users/registration_involved/case_4_assign_taks_job_users.js:44]: DRY: 3 places with identical code. Move this to utils or smth

File: tests/cypress/integration/actions_users/issue_1810_login_logout.js [hunk 3]
```diff
@@ -72,25 +85,7 @@ context('When clicking on the Logout button, get the user session closed.', () =
...
-        it('Correct user and incorrect password', () => {
-            login(Cypress.env('user'), 'randomPassword123');
-            cy.url().should('include', '/auth/login');
-            cy.closeNotification('.cvat-notification-notice-login-failed');
-        });
-
-        it('Incorrect user and incorrect password', () => {
-            login('randomUser123', 'randomPassword123');
-            cy.url().should('include', '/auth/login');
-            cy.closeNotification('.cvat-notification-notice-login-failed');
```
Comment (vkomodey) [inline tests/cypress/integration/actions_users/issue_1810_login_logout.js:93]: Here should be invalid credentials check. I assume in our case it should be existing user and not existing wallet address or vice versa

**Review by vkomodey — CHANGES_REQUESTED** [review]

File: tests/cypress/integration/actions_tasks/registration_involved/case_39_issue_2572_rename_task.js [hunk 4]
```diff
@@ -59,19 +53,18 @@ context('Rename a task.', () => {
...
             cy.userRegistration(
-                secondUser.firstName,
-                secondUser.lastName,
-                secondUserName,
-                secondUser.emailAddr,
-                secondUser.password,
+                Cypress.env('regularUserEmail'),
+                Cypress.env('regularUserEmail'),
+                Cypress.env('regularUserWalletAddress'),
+                Cypress.env('regularUserSignedEmail'),
```
[the remark this answers is not in the document]
Comment (Dzeranov) [reply tests/cypress/integration/actions_tasks/registration_involved/case_39_issue_2572_rename_task.js:61]: Their first user was an admin. I decided to name a user as 'regularUser'. So no problems here.

**Pull request merged**